src/ZoneServer/World/Actors/Characters/Character.cs

```diff
@@ -260,6 +261,11 @@ public int ClassLevel
 		/// </summary>
 		public bool IsDead => (this.Hp == 0);
 
+		/// <summary>
+		/// Returns the character's list of placed traps
+		/// </summary>
+		public List<Mob> PlacedTraps { get; set; } = new List<Mob>();
```

Comment:
You should never expose a non-thread-safe list in a multi-threaded environment. If anybody decides to suddenly modify this list from somewhere, or it just happens accidentally, you'll get exceptions. And since you're using the list in async code that doesn't have try/catches, your server will crash if that happens.